datalad_hirni/tests/utils.py

```diff
@@ -0,0 +1,21 @@
+from datalad import cfg
+
+
+def install_demo_dataset(ds, path, recursive=False):
```

Comment:
So is this is the fix ? Can't figure it out given amount of description here or in the PR

Reply:
Part of it. Will close this PR in favor of:
https://github.com/datalad/datalad-extensions/pull/12
and
https://github.com/psychoinformatics-de/datalad-hirni/pull/160